libs/execution_context/ebpf_program.h

```diff
@@ -75,13 +76,25 @@ extern "C"
      *  extension.
      *
      * @param[in] program Program that loaded the extension.
-     * @param[out] program_info_data Pointer to the program info.
+     * @param[out] program_info Pointer to the output allocated program info. Must be freed by caller by calling
+     * ebpf_program_free_program_info_data().
      * @retval EBPF_SUCCESS The operation was successful.
+     * @retval EBPF_INVALID_ARGUMENT One or more arguments are invalid.
+     * @retval EBPF_ARITHMETIC_OVERFLOW An arithmetic overflow has occured.
+     * @retval EBPF_NO_MEMORY Output program info could not be allocated.
      * @retval EBPF_ERROR_EXTENSION_FAILED_TO_LOAD The program info isn't
      *  available.
      */
     ebpf_result_t
-    ebpf_program_get_program_info_data(const ebpf_program_t* program, const ebpf_extension_data_t** program_info_data);
+    ebpf_program_get_program_info_data(_In_ const ebpf_program_t* program, _Outptr_ ebpf_program_info_t** program_info);
```

Comment:
```suggestion
    ebpf_program_get_program_info(_In_ const ebpf_program_t* program, _Outptr_ ebpf_program_info_t** program_info);
```
make function name match output structure type.  Also fix lines 75, 92, and 97

Reply:
done.